src/components/crownConfig.vue

```diff
@@ -67,6 +68,11 @@ export default {
         'processmaker-modeler-pool',
         'processmaker-modeler-intermediate-message-catch-event',
       ],
+      invalidSequenceFlowSources: [
```

Comment:
@tuantruong625 Could add 'processmaker-modeler-sequence-flow' to constants.js for the sequence flow conversion.